website/versioned_docs/version-0.80.0/binary-analysis.md

```diff
@@ -27,7 +27,7 @@ df.select($"url", $"filename", $"extension", $"mime_type_web_server", $"mime_typ
 
 Will extract all following information from images in a web collection:
```

Comment:
sorry, I should have included line # to make things easier - "audio files" should replace "image" in line 27